src/main/java/org/example/bookshop/service/impl/UserServiceImpl.java

```diff
@@ -31,6 +34,11 @@ public UserResponseDto register(UserRegistrationRequestDto requestDto) {
         User user = userMapper.toModel(requestDto);
         user.setPassword(passwordEncoder.encode(user.getPassword()));
         user.setRoles(Set.of(roleRepository.findByRole(Role.RoleType.USER)));
-        return userMapper.toDto(userRepository.save(user));
+        user = userRepository.save(user);
+
+        ShoppingCart shoppingCart = new ShoppingCart();
```

Comment:
create method `createShoppingCart(User user)` in ShopCartService and implement the logic